src/mono/mono/mini/mini-runtime.c

```diff
@@ -5555,3 +5555,28 @@ mono_invoke_runtime_init_callback (void)
 		mono_atomic_xchg_i64 ((volatile gint64 *)&runtime_init_thread_id, (gint64)G_MAXUINT64);
 	}
 }
+
+gboolean
+jit_call_can_be_supported_by_interp (MonoMethod *method, MonoMethodSignature *sig, gboolean is_llvm_only)
+{
```

Comment:
This should have a mono_ or mini_ prefix.

Reply:
Modified.